package.json

```diff
@@ -57,15 +59,14 @@
     "cross-env": "^7.0.3",
     "dotenv": "^16.4.1",
     "ethers": "^6.14.0",
-    "ethrpc-gateway": "../web3-jsonrpc-gateway",
+    "ethrpc-gateway": "^2.2.0",
     "json-bigint": "^1.0.0",
     "lodash.merge": "^4.6.2",
-    "witnet-solidity-bridge": "latest",
-    "@witnet/sdk": "^1.0.0"
+    "witnet-solidity-bridge": "~2.1.9",
+    "witnet-wrapped-wit": "^1.0.0",
+    "@witnet/sdk": "^1.0.6"
```

Comment:
Package managers add packages in alphabetical order. Let's keep them at the top to avoid conflicts

Reply:
Isn't it @ < a ?